frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostStatus.tsx

```diff
@@ -48,10 +53,35 @@ const BareMetalHostStatus: React.FC<StatusProps> = ({ status, title, ...props })
       return <AddDiscoveredHostButton host={props.host} />;
     case [NODE_STATUS_STARTING_MAINTENANCE, NODE_STATUS_UNDER_MAINTENANCE].includes(status):
       return <MaintenancePopover title={statusTitle} maintenance={props.maintenance} />;
+    case status === HOST_STATUS_INSPECTING:
+      return (
+        <ProgressStatus title={statusTitle}>
+          The hardware details for the host are being collected by an agent. This operation will
```

Comment:
```suggestion
          The hardware details of the host are being collected. This will take a while. The host will become available when finished.
```